api/models/Comment.js

```diff
@@ -26,7 +26,8 @@ module.exports = bookshelf.Model.extend(Object.assign({
     return this.hasMany(Thank)
   },
 
-  community: function () {
+  community: async function () {
```

Comment:
could there be any weird side effects of making this call async? did you look around the code in the other places it is used?

Reply:
I had, yes. It appears, and clicks for me that, that Comment#community is actually not used anywhere else, but bookshelf is tricky and I couldn't guarantee it. Finger tested things. If we have a staging backend/frontend going it'd be a good thing, but the testing I did against the current Production snapshot came-up clean.

Reply:
Cool. Staging is up and running so I can test there